src/commcare_cloud/commands/ansible/ops_tool.py

```diff
@@ -437,3 +440,17 @@ def _get_lines(proc):
         return str(color_error('timeout\n')), []
     else:
         return None, out.splitlines()
+
+
+def chunked(iterable, n, fillvalue=None):
+    """
+    Collect data into fixed-length chunks
+
+    >>> list(chunked('ABCDEFG', 3, 'x'))
+    [('A', 'B', 'C'), ('D', 'E', 'F'), ('G', 'x', 'x')]
+
+    """
+    # Pulled shamelessly from itertools recipes
+    # https://docs.python.org/3/library/itertools.html#itertools-recipes
+    args = [iter(iterable)] * n
```

Comment:
so sneaky!

Reply:
Yeah! I would not have thought of that. :nerd_face: